### DISCUSSION TITLE
"Beaten" game, NEW trophy award level

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#687] [Ideas]
**Beaten** would be added as a new trophy type level.

**Beaten** would be the trophy level under **Completed** and **MASTERED** 

As for border colors for the trophy, **Beaten** would have none, **Completed** would have a blue border to match softcore colors, and **MASTERED** would remain unchanged.

**Beaten** would only work on Hardcore, not softcore.

For this to work there would need to be a change made for the dll as well, which would allow tagging a single achievement or a group of achievements for the game to be considered **Beaten**.

The goal is here to give players another more reasonable milestone, and have a way for them to show off games they like.

**Comment by julkasalsa** [comment]
I was just about to bring the same thing ;p I think we should avoid the word "softcore" officialy. the best would be not bringing the game "Beating" award to softcore, because. I insist we would use "Beaten", becasue some players are already accustomed to it becasue it's used as similiar term in Backloggery (B-Beaten, C-Completed, M-Mastered). Additionaly we could use Hardcore only as a point multiplier, and award game "Mastered' award for completing the Bonus content, which should be tied to the same game entry.

**Comment by kdecks** [comment]
I'm fine with your terminology, and I'm editing the post to match it. I'm not fine with the idea of mastering sets requiring bonus content. Most bonus content superfluous.

I'm also bumping this in hopes that someone who has the skills can put it into action! To me this is the most wanted feature right now.

**Comment by Jamiras** [comment]
> For this to work there would need to be a change made for the dll as well, which would allow tagging a single achievement or a group of achievements for the game to be considered Beaten.

It seems like this could be integrated into the "reorder achievements" page without requiring DLL changes. If we made it part of the DLL, we'd also have to make it part of the API, and flagging (or unflagging) an achievement would cause the Last Updated timestamp to be modified.

**Comment by rzumer** [comment]
I disagree with this idea, because many games cannot be "beaten", which means that it is adds significant complexity for developers and maintainers without a clear benefit, and will inevitably lead to disagreements and conflicts.

**Comment by julkasalsa** [comment]
Most of games still can be beaten, and for others there is a point when they either are looping in, becoming too hard, there is nothing left to do except repeating the same thing or getting all optional stuff.  Nonetheless I don't think this matter could bring any serious issues. Most of active players are focusing on getting side awards, and actual achievements are relegated to the role of steps. More than one award possible to earn might reduce this problematic focus.

**Comment by rzumer** [comment]
That most games have a point that can be defined as completion doesn't quite cut it. Have I "beaten" Donkey Kong if I complete Game A? Or do I have to do Game B as well? What about the 2-player versions, do they not count? If a game loops, but has slightly different even loops, do I have to finish one or two loops? There is no single answer, and making developers choose will be yet another contentious issue to worry about.

And how will it be handled on the site? Will it be an optional "award"? What if the rules for completion don't map to a game's achievements? How will disagreements be resolved? How will revisions that affect the rules of the award be handled? Again, more problems without clear solutions, for a "nice to have" feature.

The only way that I can see these problems being avoidable is by making this fully optional, not a developer responsibility, and *prohibiting* set modifications for the purpose of implementing a "Beaten" award. However, in that case, the question of whether this is a useful feature, let alone one that is worth the implementation and maintenance effort, remains.

With that considered, at this point, I would not label this as "To Do", but as a very, very low priority. There are many problems that should be resolved on many levels (technical, usability, developer and moderator tooling, etc.) before we can afford to consider features that will introduce significant complexity on each of those levels for questionable benefit.

>Most of active players are focusing on getting side awards, and actual achievements are relegated to the role of steps. More than one award possible to earn might reduce this problematic focus.

This is not a solution. People chasing achievements won't stop until they have them all, and that won't change no matter how many participation awards you stack on top of that. Some might be convincing themselves that they would stop with an intermediary completion award, but it's unlikely that they would. It's addictive behavior.

**Comment by wescopeland** [comment]
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1714.
